app/delivery/send_to_providers.py

```diff
@@ -224,8 +224,8 @@ def get_html_email_options(service):
 
     if service.email_branding is None:
         return {
-            'govuk_banner': True,
-            'brand_banner': False,
+            'fip_banner_english': True,
```

Comment:
Let's add a comment to spell out FIP - again for folks who won't know what that is.